CHANGELOG.md

```diff
@@ -1,5 +1,82 @@
 # PowerShell Preview Extension Release History
 
+## v2020.2.0
+### Thursday, February 20, 2020
+
+#### Release of preview work to stable branch
+
+This release, coinciding with the [GA release of PowerShell 7](link to PowerShell 7 release),
+brings a year of work on the PowerShell extension into the stable release.
+The timing of this release is deliberate, since some of the new features
+depend on additions and bugfixes in PowerShell 7,
+and even those that don't generally work better in PowerShell 7
+thanks to many improvements shipping with it.
+
+Some changes that stabilise in this release include:
+

```

Comment:
I think the use of "stabilise" is concise but also could be slightly confusing? 
Maybe something like "Some changes that will now be available through the stable release channel include:"